swagger_server/controllers/survey_response.py

```diff
@@ -127,6 +137,39 @@ def _get_collection_exercise(self, collection_exercise_id):
             ons_env.logger.debug('Collection Exercise not found')
         return collection_exercise
 
+
+    def _get_survey_service(self, survey_id):
+        """
+        Used to get the SurveyRef number from the survey service. This is used to construct the file name of an
+        uploaded survey.
+        See: https://github.com/ONSdigital/rm-survey-service/blob/master/API.md
+
+        :param survey_id: The survey_id UUID to search with
+            e.g. http://localhost:8080/surveys/cb0711c3-0ac8-41d3-ae0e-567e5ea1ef87
+
+        :return: JSON e.g.
+
+        {
```

Comment:
A bit too verbose for me

Reply:
please leave this in. it's a god send in years to come. please read: >>> import this